docs/source/interactive/autoawait.rst

```diff
@@ -13,8 +13,8 @@ ability to run asynchronous code from the REPL. Constructs which are
 :exc:`SyntaxError` s in the Python REPL can be used seamlessly in IPython.
 
 The example given here are for terminal IPython, running async code in a
-notebook interface or any other frontend using the Jupyter protocol will need to
-use a newer version of IPykernel. The details of how async code runs in
+notebook interface or any other frontend using the Jupyter protocol need to
```

Comment:
either "... need**s** IPykernel..." (removing the "to") or "... need**s** to **use** IPykernel" if you want to keep the "to" in there. Also, while we're making a change here, could you change the line above (15) to read "The example**s** given here are..."